tmt/steps/discover/fmf.py

```diff
@@ -276,12 +321,6 @@ class DiscoverFmf(tmt.steps.discover.DiscoverPlugin[DiscoverFmfStepData]):
     in git since a given revision. This can be particularly useful when
     testing changes to tests themselves (e.g. in a pull request CI).
 
```

Comment:
What about keeping just the list of relevant options? Something like:

```suggestion
Related keys: ``modified-only``, ``modified-url``, ``modified-ref``
```

Reply:
Key list included !

Reply:
Thanks.